File: 문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int p_count = 0;
    int y_count = 0;

    for (int i = 0; i < s.size(); i++) {
        if (s[i] == 'p')
            p_count++;
        else if (s[i] == 'P')
            p_count++;
        else if (s[i] == 'y')
            y_count++;
        else if (s[i] == 'Y')
            y_count++;
    }
    if (p_count != y_count)
        answer = false;
    else
        answer = true;

    cout << "Hello Cpp" << endl;

    return answer;
}   

File: Lv.1/Python/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    p_cnt = 0
    y_cnt = 0
    for i in s:
        if i == 'p' or i=='P':
            p_cnt+=1
        elif i == 'y' or i =='Y':
            y_cnt +=1
    if p_cnt == y_cnt:
        answer = True
    else: answer = False
    return answer